### ISSUE TITLE
Facility locations: map thumbnail needs better alt text

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#18975]
## Description
Maps to collab cycle feedback from regional offices: https://github.com/department-of-veterans-affairs/va.gov-team/issues/87425

In the "Location and contact information" section for all facilities, the image of a map has alt text that just says "Static map."

When an image is the only content of a link, it's alt text needs to communicate both what the image is and where the link is going. As currently coded, only the image is described --- no information on where the link takes you.

### VBA
e.g https://www.va.gov/portland-va-regional-benefit-office/

<details><summary>Markup screenshot</summary>

![Screenshot 2024-08-16 at 4 42 32 PM](https://github.com/user-attachments/assets/53a8a445-0851-41ab-a956-367900857c66)
</details>

### Vet Center
https://www.va.gov/portland-or-vet-center/

<details><summary>Markup screenshot</summary>

![Screenshot 2024-08-16 at 4 44 53 PM](https://github.com/user-attachments/assets/3fa47948-5386-4703-b067-c73f0ed2f54d)
</details>

### VAMC
https://www.va.gov/minneapolis-health-care/locations/minneapolis-va-medical-center/

<details><summary>Markup screenshot</summary>

![Screenshot 2024-08-16 at 4 43 35 PM](https://github.com/user-attachments/assets/826741b9-c0e6-442c-8c48-20e8f4c47743)
</details>

### Engineering notes / background
alt text is set here: https://github.com/department-of-veterans-affairs/vets-website/blob/218bafb1d624cc6b31dcf15fe4a53cb33ab9e709/src/applications/static-pages/facilities/FacilityMapWidget.jsx#L77

## Acceptance criteria
- [ ] Image alt text should include ...? and be verified for locations page maps on
  - [ ] VBA
  - [ ] VAMC
  - [ ] Vet Centers
- [ ] When verified, also close https://github.com/department-of-veterans-affairs/va.gov-team/issues/87426

**Comment by jilladams** [comment]
@laflannery I looked at the statement "alt text needs to communicate both what the image is and where the link is going. As currently coded, only the image is described --- no information on where the link takes you" and am not quite certain what the AC should be here. The image itself is inside of a link that has an HREF to the map. Could you help inform AC?

![Screenshot 2024-08-16 at 5 00 30 PM](https://github.com/user-attachments/assets/5055071e-55e9-43d6-b37c-4cac01ee3901)

**Comment by jilladams** [comment]
Nevermind - found https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18268, collapsing this into that one.
